app/src/main/java/org/mozilla/fenix/home/recenttabs/RecentTabsListFeature.kt

```diff
@@ -29,20 +27,12 @@ class RecentTabsListFeature(
 ) : AbstractBinding<BrowserState>(browserStore) {
 
     override suspend fun onState(flow: Flow<BrowserState>) {
-        flow.map { it.selectedTab }
-            .ifAnyChanged { arrayOf(it?.id, it?.content?.title, it?.content?.icon) }
```

Comment:
We don't want to remove the observers of the title/icon to prevent a regression of this bug: https://github.com/mozilla-mobile/fenix/issues/20054

Reply:
Thank you!
Will then observe `selectedNormalTab?.id`, `selectedNormalTab?.content?.title`, `selectedNormalTab?.content?.icon` and `normalTabs`.